### ISSUE TITLE
training_loss doesn't change

### ISSUE THREAD

**Issue by csanadpoda** [clovaai/donut#177]
I'm trying to pretrain Donut on Hungarian, and I'm following the steps described in the documentation. I've successfully used SynthDOG to generate a dataset, and I'm running the provided script (`python train.py --config config/base.yaml --exp_version "base"`
), but the training loss never decreases, it just stays the same:
![image](https://user-images.githubusercontent.com/17254090/229632690-5156615b-a447-4d0e-b2cd-e2e740a4dffe.png)

I have no idea what I'm doing wrong. I did need to port this to PyTorch 2.0 and lightning 2.0 bc my GPU wasn't supported on the versions you provided, but I've only needed to make minor adjustment and I followed the upgrade guides. What information do you need to help, how can I make sure learning actually happens?

**Comment by Wyzix33** [comment]
Try it on colab to see if it works.

**Comment by csanadpoda** [comment]
> Try it on colab to see if it works.

I actually tried it in WSL and it works.... Seems to be in issue with either Windows 11 or the GLOO solver, since everything else stayed the same.

**Comment by 0chandansharma** [comment]
num_workers: 0 

works for me

**Comment by csanadpoda** [comment]
> num_workers: 0

That would mean significant performance loss though.

**Comment by FreestyleMove** [comment]
> num_workers: 0

> works for me

thanks bro，You are genius. I’m tring to fix the preblom that spent my about daytime. its so wired. Doesn't the para(num_workes) mean is how  many cpu cores will be used?
